app/controllers/items_controller.rb

```diff
@@ -547,7 +547,7 @@ def set_rights
   end
   # set the content type in the content metadata
   def set_content_type
-    unless %w(book file image map manuscript).include? params[:new_content_type]
+    unless Constants::CONTENT_TYPES.include? params[:new_content_type]
       render :status => :forbidden, :text => 'Invalid new content type.'
```

Comment:
Not sure... the intent here is that if someone updated the constant they would also need to update the spec. Making sure that someone didn't remove one and know that it was actually expected to be there.